<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>50</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>30</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Xn</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>113</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="groupBox_7">
      <property name="minimumSize">
       <size>
        <width>190</width>
        <height>50</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Order</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="minimumSize">
       <size>
        <width>190</width>
        <height>50</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>nPoints</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>230</width>
        <height>110</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>17</y>
         <width>145</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>30</height>
        </size>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
       </property>
       <item>
        <property name="text">
         <string>Sine integral</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cosine integral</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bessel Function</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fresnel Integral S</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fresnel Integral C</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>530</width>
        <height>380</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCustomPlot" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>650</width>
           <height>280</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>70</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>50</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>X0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>113</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="minimumSize">
       <size>
        <width>550</width>
        <height>100</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="connectButton">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="computeButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Compute</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="breakButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Break</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_5</zorder>
   <zorder>groupBox_6</zorder>
   <zorder>groupBox_7</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QFrame</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
